plugins/modules/packaging/os/pacman.py

```diff
@@ -52,7 +52,10 @@
 
     executable:
         description:
-            - Name of binary to use. This can either be C(pacman) or a pacman compatible AUR helper.
+            - Path of the binary to use. This can either be C(pacman) or a pacman compatible AUR helper.
+            - Pacman compatibility is unfortunately ill defined, in particular, this modules makes
+              extensive use of the ``--print-format`` directive which is known not to be implemented by
+              some AUR helpers. (Notably, ``yay``)
```

Comment:
```suggestion
              extensive use of the C(--print-format) directive which is known not to be implemented by
              some AUR helpers (notably, C(yay)).
```

Reply:
that's hilarious, I just fixed the formatting in #4316, yet I wrote it wrong here...